src/NUnitFramework/framework/Internal/Results/TestSuiteResult.cs

```diff
@@ -220,74 +220,60 @@ public override IEnumerable<ITestResult> Children
         public virtual void AddResult(ITestResult result)
         {
 #if PARALLEL
-            var childrenAsConcurrentQueue = Children as ConcurrentQueue<ITestResult>;
-            if (childrenAsConcurrentQueue != null)
-                childrenAsConcurrentQueue.Enqueue(result);
```

Comment:
I don't understand this refactoring is being done. Is this PR fixing more than one thing at a time?